app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java

```diff
@@ -187,6 +189,9 @@ private void btnLendBook(View view)
         //TODO: launch scan
         this.bookDocument.update(getString(R.string.status), getString(R.string.bPending));
         onBackClick(view);
+
+        // Send In-app and Push notification to Borrower
+        sendNotification(selectedBook.getRequesters().get(0), getString(R.string.my_requests_pick_up_location_set_message));
```

Comment:
Thank you for adding the other notification, but this one is technically wrong. The owner sets the location using the other button. If you want a notification here it would be something like "Owner scanned book to confirm lend"

Reply:
Oh right, that's better. Does "Book confirmed to lend by: owner" work?